data/mods/Aftershock/vehicles/afs_vehicle_parts.json

```diff
@@ -161,6 +176,41 @@
       { "item": "press", "charges": 0, "prob": 50 }
     ]
   },
+  {
+    "type": "vehicle_part",
+    "id": "afs_control_station",
```

Comment:
I'd be fine with seeing this in mainline and expanded to a couple of other military vehicles.

I'd also like a version of this without REMOTE_CONTROLS - call it an integrated control console - added to all the military vehicles in Tanks, if that's something you would be interested in.

Reply:
I can put it in mainline, but I'd also open it as a different PR so as not to muddy the waters. I've never actually played using Tanks, but I don't think it should be too much of a stretch to put it in.